### PR TITLE
Clarify move_group tolerance setting

### PR DESCRIPTION
**Pull request by methylDragon**
Relevant issue: https://github.com/ros-planning/moveit/issues/1475

Adds a clarification of the setGoalTolerance() method for move_group

### CODE REVIEW

File: doc/move_group_interface/move_group_interface_tutorial.rst [hunk 1]
```diff
@@ -52,3 +52,12 @@ The entire code can be seen :codedir:`here in the MoveIt GitHub project<move_gro
 The Launch File
 ---------------
 The entire launch file is :codedir:`here<move_group_interface/launch/move_group_interface_tutorial.launch>` on GitHub. All the code in this tutorial can be run from the **moveit_tutorials** package that you have as part of your MoveIt setup.
+
+
+A Note on Setting Tolerances
+---------------
+If you looks through the `move_group documentation<http://docs.ros.org/lunar/api/moveit_ros_planning_interface/html/classmoveit_1_1planning__interface_1_1MoveGroupInterface.html>` you might find a nifty method called `setGoalTolerance()` or related ones that might lead you to think that you are able to configure the **execution** tolerances for your robotic manipulator.
```
Comment (rhaschke) [inline doc/move_group_interface/move_group_interface_tutorial.rst]:
```suggestion
If you look through the `move_group documentation<http://docs.ros.org/lunar/api/moveit_ros_planning_interface/html/classmoveit_1_1planning__interface_1_1MoveGroupInterface.html>` you might find a nifty method called `setGoalTolerance()` or related ones that might lead you to think that you are able to configure the **execution** tolerances for your robotic manipulator.
```

Comment (rhaschke) [reply doc/move_group_interface/move_group_interface_tutorial.rst]: Don't refer to a Lunar documentation when submitting to master.

File: doc/move_group_interface/move_group_interface_tutorial.rst [hunk 2]
```diff
@@ -52,3 +52,12 @@ The entire code can be seen :codedir:`here in the MoveIt GitHub project<move_gro
 The Launch File
 ---------------
 The entire launch file is :codedir:`here<move_group_interface/launch/move_group_interface_tutorial.launch>` on GitHub. All the code in this tutorial can be run from the **moveit_tutorials** package that you have as part of your MoveIt setup.
+
+
+A Note on Setting Tolerances
+---------------
```
Comment (rhaschke) [inline doc/move_group_interface/move_group_interface_tutorial.rst]: Please adjust length of heading marker.

File: doc/move_group_interface/move_group_interface_tutorial.rst (hunk 1 above)
Comment (methylDragon) [reply doc/move_group_interface/move_group_interface_tutorial.rst]: Which version of documentation should I be using? Melodic?

Comment (BryceStevenWilley) [reply doc/move_group_interface/move_group_interface_tutorial.rst]: Use `docs.ros.org/latest` instead.

**Comment by rhaschke** [comment]
> What's going on over at `moveit_ci`?

This is related to https://github.com/ros-planning/moveit_ci/pull/74 @davetcoleman requested. Looks like we didn't modified all variables yet :wink: I will have a look...

**Comment by rhaschke** [comment]
Travis is fixed.

**Pull request merged**